package.json

```diff
@@ -18,25 +18,29 @@
     "bitfield": "^1.0.2",
     "capture-frame": "^1.0.0",
     "chromecasts": "^1.8.0",
+    "cp-file": "^3.2.0",
     "create-torrent": "^3.24.5",
     "debounce": "^1.0.0",
     "deep-equal": "^1.0.1",
     "dlnacasts": "^0.1.0",
     "drag-drop": "^2.12.1",
     "electron": "1.4.1",
     "es6-error": "^3.0.1",
-    "fs-extra": "^0.30.0",
+    "fn-getter": "^1.0.0",
     "iso-639-1": "^1.2.1",
     "languagedetect": "^1.1.1",
     "location-history": "^1.0.0",
+    "lodash.merge": "^4.6.0",
```

Comment:
We don't actually use this. It's there so we can use it instead of `lodash/merge` (see comments at top of `renderer/main.js` for explanation)